zookeeper-server/src/main/java/org/apache/zookeeper/server/FinalRequestProcessor.java

```diff
@@ -212,341 +209,343 @@ public void processRequest(Request request) {
                 throw ke;
             }
 
-            LOG.debug("{}",request);
+            LOG.debug("{}", request);
 
             if (request.isStale()) {
                 ServerMetrics.getMetrics().STALE_REPLIES.add(1);
             }
 
             switch (request.type) {
-            case OpCode.ping: {
-                lastOp = "PING";
-                updateStats(request, lastOp, lastZxid);
+                case OpCode.ping: {
```

Comment:
There's a lot of churn in the PR in this switch statement but as far as I can see, no logical changes.

Reply:
It is done by an auto formatting phase. For introduce such a phase many of '+'/'{' should be around by space could be fixed. It is a format related change and sure, no logical changes.